versions.tf

```diff
@@ -5,7 +5,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "2.62.0"
+      version = "3.20.0"
```

Comment:
They're up to 3.21.1 now. Let's make sure we can support close to the latest version as we keep moving forward.